app/jinja_filters.py

```diff
@@ -82,8 +82,21 @@ def format_address_list(user_entered_address=None, metadata_address=None):
     return address_list
 
 
-def format_unit(unit, value=''):
-    return units.format_unit(value=value, measurement_unit=unit, length='short', locale=DEFAULT_LOCALE)
+def format_unit(unit, value='', length='short'):
+    return units.format_unit(value=value, measurement_unit=unit, length=length, locale=DEFAULT_LOCALE)
+
+
+def format_unit_input_label(unit, length='short', use_plural_for_label=False):
```

Comment:
Should we not default `use_plural_for_label` to `True`?

Reply:
and probably rename to `use_plural` given the context of label is in the function name?

Reply:
To your first question, I don't know.  What are we more likely to want?  Everything so far has been singular til now so I assumed that should be the default, though it's not much effort to change it.

To your second point, that's a good point!

Reply:
On second thought, I think the default should be short/singular, because of the overwhelming amount of stuff that has its labels in that form.  Do you reckon that'll change?  If not, then I think we should leave it as is